src/main/java/mate/academy/model/MovieSession.java

```diff
@@ -0,0 +1,71 @@
+package mate.academy.model;
+
+import java.time.LocalDateTime;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToOne;
+
+@Entity
+public class MovieSession {
```

Comment:
It is better to name the tables with a lowercase letter and separate the words with an underline